app/controllers/user.js

```diff
@@ -87,3 +88,45 @@ exports.userList = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.singUpAdmins = async (req, res, next) => {
+  const auth = req.headers[AUTHORIZATION];
+
+  const plainText = decoder(auth);
+
+  const requester = await User.getUserBy(plainText.email);
+
+  if (requester.permission === 'administrator') {
```

Comment:
Do this in a middleware for example, put enum values in a constant file or something like that